### PR TITLE
Update styling docs

### PR DESCRIPTION
**Pull request by RoyStegeman**

### CODE REVIEW

File: doc/sphinx/source/contributing/rules.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
-```eval_rst
 .. _rules:
-```
+
```
Comment (scarlehoff) [inline doc/sphinx/source/contributing/rules.md:2]: why did you remove this?

**Review by scarlehoff — APPROVED** [review]
Thank you very much!

File: doc/sphinx/source/contributing/rules.md (hunk 1 above)
Comment (RoyStegeman) [reply doc/sphinx/source/contributing/rules.md:4]: Was considering changing .md to .srt but then decided that is too much effort.
